lib/src/checkbox/checkbox.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+class MUICheckBox extends StatefulWidget {
```

Comment:
Hey could you please add color options to the checkbox like selected color, not selected color, border color, instead of relying on Themes. Else everything LGTM 👍🏼

Reply:
oh, sure, forgot about that.